### DISCUSSION TITLE
The name style for anonymous select item

### DISCUSSION THREAD

**Discussion by xudong963** [databendlabs/databend#5053] [General]
Consider the following sqls
```sql
create table t(a int);
insert into t values(1),(2);
select sum(a) from t group by a;
```
For our old planner: the output is the following, the alias name is mysql style
```
sum(a)
1
2
```
For our new planner: the alias name will be pg style, as the code shows:
```rust
pub fn get_expr_display_string(expr: &Expr) -> String {
    match expr {
        Expr::ColumnRef { column, .. } => column.name.clone(),
        _ => {
            // TODO: this is Postgres style name for anonymous select item
            "?column?".to_string()
        }
    }

```
This ticket is to discuss which style we will end up with?

**Comment by leiysky** [comment]
cc @sundy-li @BohuTANG @zhang2014

**Reply by sundy-li** [reply]
I prefer to pass various dialects here to cover both names or just use MySQL style.

**Reply by leiysky** [reply]
Agree. I think MySQL style is more readable.

**Reply by zhang2014** [reply]
Agree.

**Reply by xudong963** [reply]
Agree

**Reply by Xuanwo** [reply]
Hey, let's use reactions instead of plain `Agree` :laughing: 

I will hide all replies without new information attached.

**Comment by xudong963** [comment]
Mysql style passed unanimously 😂
